tests/conftest.py

```diff
@@ -76,7 +76,9 @@ def relocatable_cluster(test_dir, test_id):
         'range_request_timeout_in_ms': timeout,
         'write_request_timeout_in_ms': timeout,
         'truncate_request_timeout_in_ms': timeout,
-        'request_timeout_in_ms': timeout
+        'request_timeout_in_ms': timeout,
+        'skip_wait_for_gossip_to_settle': 0,
+        'ring_delay_ms': 0
```

Comment:
This seems to do more than refactoring.
Better extract it to a separate patch and document the motivation

Reply:
I've added this so I can run the unittest quicker, (we can't enable it by default in dtest, but those integration test, this probably good enough)